src/ui/components/MapBox/MapBox.js

```diff
@@ -0,0 +1,252 @@
+ui.class.MapBox = class MapBox extends HTMLElement {
+	constructor(props) {
+		super();
+		const self = this;
+
+		let m_divTag;
+		let m_map;
+		let m_parentTag;
+		let m_props;
+		let m_userLocation;
+		let m_zoomDefault = 0.01;
+
+		this.addPoints = addPoints;
+		this.setup = setup;
+
+		if (props) {
+			setup(props);
+		}
+
+		function addPoints(points) {
```

Comment:
Move function addPoints{} to end of file. The top of the file is reserved for the functions that execute first and in cronological order

Reply:
Done!